src/amo/components/Header/index.js

```diff
@@ -61,7 +61,7 @@ export class HeaderBase extends React.Component {
       <Link className="Header-title" to="/">
         <span className="visually-hidden">
           {// translators: "Firefox" should not be translated. :-)
-          i18n.gettext('Firefox Add-ons')}
+          i18n.gettext('Firefox Browser Add-ons')}
```

Comment:
I didn't see anything in the issue or the bug specifically about making this change, but I guess it was mentioned elsewhere?

Reply:
I mentioned in the notes to QA so they're aware of it. This text is an accessible version of the wordmark and since this is what the wordmark says now, I updated it. The wordmark was presented to us by UX in https://bugzilla.mozilla.org/show_bug.cgi?id=1582891